CHANGELOG.md

```diff
@@ -3,6 +3,146 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.3.0
+
+_Jan 21, 2022_
+
+A big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Allow to group rows based on column value (#3277) @flaviendelangle
+
+  ⚠️ This feature is temporarily available on the Pro plan until the release of the Premium plan.
+
+  To avoid future regression for users of the Pro plan, the feature needs to be explicitly activated using the rowGrouping experimental feature flag.
+
+  ```tsx
+  // To fully control
+  <DataGridPro
+    rowGroupingModel={rowGroupingModel}
+    onRowGroupingModel={newModel => setRowGroupingModel(newModel)}
+    experimentalFeatures={{ rowGrouping: true }}
+  />
+
+  // To initialize without controlling
+  <DataGridPro
+    initialState={{
+      rowGrouping: {
+        model: rowGroupingModel,
+      },
+    }}
+    experimentalFeatures={{ rowGrouping: true }}
+  />
+  ```
+
+  See the [documentation](https://mui.com/components/data-grid/group-pivot/#row-grouping) for more details.
+
+- ⚡ Add `is any of` filter operator (#2874) @alexfauquette
+
+  The new filter operator `is any of` allows the user to provide multiple values. It opens access to complex filtering pattern mixing `AND` and `OR` logic connectors, such as `status is any of filled or rejected, and currency is any of EUR or USD`.
+
+  <img src="https://user-images.githubusercontent.com/45398769/150486348-996a938f-db24-426f-bfe3-c06337f71807.gif" width="964" height="657">
```

Comment:
```suggestion
  <img src="https://user-images.githubusercontent.com/45398769/150486348-996a938f-db24-426f-bfe3-c06337f71807.gif" width="770" height="370">
```

The size looks wrong :)

![Screenshot 2022-01-21 at 10 20 43](https://user-images.githubusercontent.com/13808724/150500939-a84edc30-cf03-4ad7-b2bc-0465b938c50a.png)